confmap/provider/eecprovider/provider_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package eecprovider
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/collector/confmap/confmaptest"
+)
+
+func TestSupportedScheme(t *testing.T) {
+	fp := NewFactory().Create(confmaptest.NewNopProviderSettings())
+	assert.Equal(t, "http", fp.Scheme())
```

Comment:
I'm not an expert, but shouldn't this be `eec`?

Reply:
Good catch, it should be. I've updated this.